src/proper_gen_next.erl

```diff
@@ -203,6 +204,22 @@ apply_temperature_scaling(Generator) ->
       end
   end.
 
+%% Add any parameters that were added to the original generator,
+%% so that they apply in the targeted generation as well.
+
+apply_parameters(Gen, RawType) ->
```

Comment:
suggestion:
  factor out the repeated construction of the `?LAZY(Gen(Base, Temp)))` in this code?  (i.e. assign it in a variable before the `Type = ... ` line?